templates/default/nginx.yaml.erb

```diff
@@ -1,6 +1,12 @@
 instances:
 <% @instances.each do |i| -%>
   - nginx_status_url: <%= i['nginx_status_url'] %>
+    <%= "ssl_validation: #{i['ssl_validation']}" unless i['ssl_validation'].nil? -%>
```

Comment:
IMO the indentation and format of this line could break easily if we make changes to it in the future, could you rather write:
```erb
<% unless i['ssl_validation'].nil? -%>
    ssl_validation: i['ssl_validation']
<% end -%>
```

What do you think?